profiling/src/profile/internal/value_type.rs

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+use super::super::{pprof, Id, StringId};
+
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+pub struct ValueType {
+    pub r#type: StringId,
```

Comment:
Since we're making our own struct, should we call this `typ` instead?

Reply:
(I do prefer `type` personally :P )